bundle/Resources/public/js/fields-sorting.js

```diff
@@ -0,0 +1,29 @@
+(function ($) {
+    'use strict';
+    $(function () {
+        var $rows = $('.js-form-field-row');
```

Comment:
You should surround the whole thing with `$('.js-form-fields-collection-entries.).each(function(){` else it won't work when creating a new form, as there is no ".js-form-field-row"